renderer/src/features/mcp-servers/components/groups-manager/index.tsx

```diff
@@ -1,15 +1,13 @@
 import type { ReactElement } from 'react'
 import { useQuery } from '@tanstack/react-query'
 import { getApiV1BetaGroups } from '@api/sdk.gen'
-import { useRouterState } from '@tanstack/react-router'
+import { useParams } from '@tanstack/react-router'
 import { AddGroupButton } from './add-group-button'
 import { GroupList } from './group-list'
 
 export function GroupsManager(): ReactElement {
-  const router = useRouterState({ select: (s) => s.location.search }) as Record<
-    string,
-    unknown
-  >
+  const params = useParams({ from: '/group/$groupName' })
```

Comment:
does it should have a `default` name for default group?

Reply:
not sure what you mean? :thinking: